packages/terra-form-select/tests/jest/SelectField.test.jsx

```diff
@@ -18,6 +18,20 @@ it('should render a default SelectField component', () => {
   expect(shallowWithIntl(select)).toMatchSnapshot();
 });
 
+it('should render a disabled SelectField component', () => {
+  const select = (
+    <SelectField label="Label" defaultValue="blue" selectId="select-id">
```

Comment:
This snapshot example should include the `disabled` prop on the `SelectField` component.